docs/devices/SRTS-A01.md

```diff
@@ -65,9 +65,11 @@ The possible values are: `internal`, `external`.
 ### Calibrated (binary)
 Is the valve calibrated.
 Value can be found in the published state on the `calibrated` property.
-It's not possible to read (`/get`) or write (`/set`) this value.
```

Comment:
@HolgerAusB I noticed now you removed `or write (`/set`)`, but AFAIK this is valid (it is not possible to set it)

Reply:
@Koenkk I am not perfect in English. Maybe the wording is not right. You can't `/set/calibrated`, correct. But I could `/set/calibrate` to start the calibration. If that makes a difference, please change the sentence as needed.

Reply:
The `calibrate` expose was missing, I added it in https://github.com/Koenkk/zigbee-herdsman-converters/commit/2a58fbef89b13d28334c49ef6fa60f4562646b64